packages/nimble-components/docs/css-guidelines.md

```diff
@@ -213,4 +213,8 @@ export const styles = css`
 `;
 ```
 
-This utility will generate the appropriate display style, as well as a style rule to hide the host element when its `hidden` attribute is set.
\ No newline at end of file
+This utility will generate the appropriate display style, as well as a style rule to hide the host element when its `hidden` attribute is set.
+
+## Stick to enumerated values for `z-index`
+
+To avoid an unnecessary proliferation of `z-index` values (which make the code more difficult to reason about), we define a `ZIndexLevels` enum with a fixed set of values to choose from. If possible, use one of the existing values. If you instead need to establish a new stacking position relative to the existing values, create a new enum value to use. When possible, pick a value that is a power of 10.
```

Comment:
I'd vote to leave the values to dev / code owner discretion. Picking powers of 10 could result in unneccessarily big gaps and hard-to-read counting of zeros (is that 1000000 or 10000000?)

```suggestion
To avoid an unnecessary proliferation of `z-index` values (which make the code more difficult to reason about), we define a `ZIndexLevels` enum with a fixed set of values to choose from. If possible, use one of the existing values. If you instead need to establish a new stacking position relative to the existing values, create a new enum value to use.
```